Introduction (#4)
MrLongbottom/P9-Rapport: sections/introduction.tex

\section{Introduction}
%% Argue briefly for the relevance of the studied area/problem — start from general and end with your concrete problem.
A general challenge for search engines, no matter the environment, is how to acquire relevant search results from a given corpus.
While some search engines use simple algorithms, such as finding all articles including a given search term and sorting by date, this may not always give useful results.

> Because of this we want to explore ways to improve search results while using information already available from a corpus.

Reviewer: ```suggestion
Therefore we want to explore ways to improve search results while using information already available from a corpus.
```

> A way to improve searches could be by labeling documents in a corpus and using these labels to get results.

Reviewer: ```suggestion
Labeling documents in a corpus and using these labels to get relevant results could improve queries.
```

> A popular way to label documents is topic modeling.

Reviewer: ```suggestion
A popular way to label documents is through the use of topic modeling.
```

> Topic modeling gives each document in a corpus a distribution of topics, which can then be used for different applications.

Reviewer: ```suggestion
Topic modeling gives each document in a corpus a distribution of topics, which can be used for different applications.
```

More specifically, we look into \gls{lda} since it is currently the most used topic model, and is applicable for many situations\cite{lda}.

> Topic modeling does not by itself help with how to rank documents in a search, which is why an additional part is necessary.

Reviewer: ```suggestion
Topic modeling does not rank documents in a search, which is why an additional part is necessary.
```

We have chosen to explore a popular way to rank documents, which is by using PageRank.

> PageRank is an algorithm originally meant for ranking web pages in search engine results by their importance, but it can easily be extended to other types of data, such as news articles.

Reviewer: ```suggestion
PageRank is an algorithm originally used for ranking web pages in search engines, but it can easily be extended to other types of data, such as news articles.
```

A more specific implementation of PageRank is a cluster-based PageRank\cite{ClusterPageRank}.

> This method uses the similarities between documents, clusters, and a corpus as an extension to regular PageRank.

Reviewer: ```suggestion
This method uses similarities between sentences, clusters, and a corpus to create better summaries of texts.
```

> We believe this can help find further connections between documents by clustering them using the topics given by \gls{lda}.

Reviewer: ```suggestion
We believe the information propagation used in \cite{ClusterPageRank} between sentences can be applied to articles by clustering according to topics given by \gls{lda}.
```

> Our goal is thus to improve news article search by using a combination of topic modeling and PageRank.

Reviewer: ```suggestion
Our goal is thus to improve news article search by using a combination of \gls{lda} and PageRank.
```

For our experiments we use the Nordjyske database of news articles as our dataset.
The idea is to use topic modeling to apply topics to documents, and cluster the documents with similar topics.

> Then the cluster-based PageRank, which takes these clusters into account, is used in order to find relevant documents given a search query.

Reviewer: ```suggestion
The cluster-based PageRank, which takes these clusters into account, is then used to find relevant documents given a search query.
```
